ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_HasType_User()
+        {
+            //Arrange
+            Article _article = new Article();
+            User _user = new User("Tommy");
+            //Act
+            _article.Autor = _user;
+            //Assert
+            Assert.IsType<User>(_article.Autor);
+        }
+        [Fact]
+        public void CtorArticle_HasTitleAndText_TypeString()
+        {
+            //Arrange
+            Article _article = new Article();
+            //Act
+            _article.Title = "Title";
+            _article.Text = "Text";
+            //Assert
+            Assert.IsType<string>(_article.Title);
+            Assert.IsType<string>(_article.Text);
+        }
```

Comment:
This test is needless as the type of the properties are strong typezined.
You can test other variants of the constructors instead.